bindings/node.js/binding.gyp

```diff
@@ -0,0 +1,51 @@
+{
+  'targets': [
+    {
+      'target_name': 'blst-node-binding',
+      'sources': [
+        'blst_wrap.cpp',
+        '../libblst.a'
+      ],
+      # Is 'libblst.a' a library or a source?
+      'libraries': [],
+      'include_dirs': [
+        '../',
+        # '<!(node -e \"require(\'nan\')\")'
+      ],
+      
+      'cflags': [
+        '-shared',
+        '-fPIC',
+        '-std=c++11',
+        '-DBUILDING_NODE_EXTENSION',
+      ],
+      # node-gyp has both cflags_cc and cflags. Not sure which one to use
+      'cflags_cc': [
```

Comment:
Sounds like 'cflags_cc' are C++ flags and 'cflags' are C. So that above `-std-c++11` and `-DBUILDING_NODE_EXTENSION` should go here.

Reply:
`-DBUILDING_NODE_EXTENSION` appears to be added by node-gyp automatically, there is no need to bother.